01-JS-DOM-APIs/js/scripts.js

```diff
@@ -0,0 +1,140 @@
+
+window.onload = function(){
+    var sec = document.getElementById('words');
+    if(sec.style.display=='block')
+    {
```

Comment:
Try to write the if as 

```
if (sec.style.display == 'block') {
...
}
```

Is more readable.

Also use `===` instead of `==`